lib/eth/key/decrypter.rb

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2016-2022 The Ruby-Eth Contributors
```

Comment:
don't forget to add yourself to the `AUTHORS` file 🙃

Reply:
good idea :)